exchangecommon/node_management_status.go

```diff
@@ -155,11 +157,14 @@ func StatusFromNewPolicy(policy ExchangeNodeManagementPolicy, workingDir string)
 	}
 	if policy.AgentAutoUpgradePolicy != nil {
 		startTime, _ := time.Parse(time.RFC3339, policy.PolicyUpgradeTime)
+		if policy.PolicyUpgradeTime == TIME_NOW_KEYWORD {
+			startTime = time.Now()
+		}
 		realStartTime := startTime.Unix()
 		if policy.UpgradeWindowDuration > 0 {
 			realStartTime = realStartTime + int64(rand.Intn(policy.UpgradeWindowDuration))
 		}
-		newStatus.AgentUpgrade.ScheduledTime = time.Unix(realStartTime, 0).Format(time.RFC3339)
+		newStatus.AgentUpgrade.ScheduledTime = time.Unix(realStartTime, 0).String()
```

Comment:
I checked the code again, it looks like the RFC3339 is used in a lot of places. Let's switch it back here so that the output can be parsed easily by the caller.  Sorry for the inconvenience.